tests/unit/readers/cubeconfig/v1_0/test_build_friendly_error_msgs.py

```diff
@@ -270,6 +271,33 @@ def test_val_errors_both_unit_types():
     )
 
 
+def test_val_errors_invalid_uri_template():
+    """
+    Test for:-
+        TBD
+    """
+    config = Path(_test_case_dir, "invalid_uri_template.json")
+    csv = Path(_test_case_dir, "invalid_uri_template.csv")
+    cube, json_schema_validation_errors, validation_errors = _extract_and_validate_cube(
+        config, csv
+    )
+    _write_errors_to_log(json_schema_validation_errors, validation_errors)
+
+    assert isinstance(cube, Cube)
+    assert isinstance(validation_errors, list)
+    assert len(validation_errors) == 1
```

Comment:
I'd use the [`assert_num_validation_errors(...)`](https://github.com/GSS-Cogs/csvcubed/blob/879b3cb7a899a6fcb22745fa55ec59b0106d4e78/tests/unit/test_baseunit.py#L40) function here since it helps debugging what went wrong.